### PR TITLE
[Boost] Add redirect to Boost dashboard after plugin activation

### LINKED ISSUE #27390: Direct users to Boost onboarding dialogue page when activated from plugin directory
Current behavior: 
Plugin directory install > activate > stays on plugin directory

Desired behavior:
Plugin directory install > activate > boost onboarding dialogue 
[See this conversation](https://jetpackp2.wordpress.com/2022/11/09/activate-behavior-on-single-plugins/)

Video example attached

https://user-images.githubusercontent.com/49159284/201689113-0ebbd381-2ee2-48e1-b78f-7c6f3c74619a.mp4

### PR DESCRIPTION
**Pull request by dilirity**
Fixes https://github.com/Automattic/jetpack/issues/27390.

#### Changes proposed in this Pull Request:

* After activating Jetpack Boost, the user will be redirected to the Boost dashboard.

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion

p1HpG7-j3y-p2

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Checkout branch and add it as a WP plugin;
* Activate plugin from Plugins page.

### CODE REVIEW

File: projects/plugins/boost/composer.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
 		"automattic/jetpack-lazy-images": "@dev",
 		"automattic/jetpack-my-jetpack": "@dev",
 		"automattic/jetpack-plugin-deactivation": "@dev",
+		"automattic/jetpack-plugins-installer": "0.2.x-dev",
```
Comment (thingalon) [inline projects/plugins/boost/composer.json]: Based on the github actions, sounds like this needs to be `@dev` :)

**Review by thingalon — CHANGES_REQUESTED** [review]
Nice! This works well. Looks like the github actions have identified a minor technicality to fix in the composer json file but otherwise this is great.

File: projects/plugins/boost/composer.json (hunk 1 above)
Comment (dilirity) [reply projects/plugins/boost/composer.json]: Thanks! Fixed with bb5998613b72ede395457fa30c30a8b46c8f51d9.

**Review by thingalon — APPROVED** [review]
Nice :)

**Pull request merged**